rs/validator/src/ingress_validation.rs

```diff
@@ -234,7 +234,7 @@ pub enum RequestValidationError {
     InvalidRequestExpiry(String),
     #[error("Invalid delegation expiry: {0}")]
     InvalidDelegationExpiry(String),
-    #[error("The user id '{0}' does not match the public key '{:?}'", hex::encode(.1))]
+    #[error("The user id '{0}' does not match the public key '{n}'", n=hex::encode(.1))]
```

Comment:
`n` wouldn't have been my first choice for a hex-encoding variable, but it works :-)

Out of curiosity: were the changes in this file required by the new version, or you just preferred to have a variable here?

Reply:
They were required. Here you can see the authors suggestion for the migration 

https://github.com/dtolnay/thiserror/releases/tag/2.0.0

Reply:
👍 Thanks for the pointer.